test/lambda-tools-compose-container-env.test.js

```diff
@@ -0,0 +1,19 @@
+const test = require('ava');
+
+const { useLambdaContainer } = require('./helpers/lambda');
+
+const containerConfig = {
+  environment: {
+    TEST_PARAMETER: 'a test value'
+  }
+};
+
+const LAMBDA_IMAGE = 'lambci/lambda:nodejs6.10';
+
+useLambdaContainer(test, LAMBDA_IMAGE, {containerConfig});
+
+test('Compose containers can use a custom environment', async (test) => {
```

Comment:
it would be really helpful to have a test that would demonstrate how the consumers would be using this code to run 2 lambdas in the same container with different environment variables

Reply:
Ya, agree. This is a little tricky to demonstrate here though. The use case is a project that uses `docker-compose` to start the Lambda runtime container. Different test files (node processes if using `ava`) could then define different environments using `useComposeContainer()`. Previously, this was only possible using `useNewContainer()`. `useNewContainer()` provisions a new container for the test suite whereas `useComposeContainer()` re-uses the existing container. Does that make sense?

Reply:
It makes sense, thanks for the explanation